apps/omg_watcher/test/omg_watcher/web/controllers/utxo_test.exs

```diff
@@ -27,6 +26,162 @@ defmodule OMG.Watcher.Web.Controller.UtxoTest do
   @eth OMG.Eth.RootChain.eth_pseudo_address()
   @eth_hex Encoding.to_hex(@eth)
 
+  describe "standard_exitable" do
```

Comment:
agree on this, this is overtested. If you want useful controllers tests here, do just these:

- "no utxos are returned for non-existing addresses" (unhappy path handling, just in case, but really I think this isn't necessary)
- "badly encoded address works well when getting exitable utxos"

none of these requires any `OMG.DB` setup.

And if you absolutely want to test that those two endpoints behave the same, do a single test that compares the result of `get_utxos` with `get_exitable_utxos` and do the `OMG.DB` setup there only (and don't use internals, see other comment).

**But** that would be much nicer to solve with a single line in an existing integration test (which I suggested adding and is missing still...)